src/wasm-lib/kcl/src/std/args.rs

```diff
@@ -1688,3 +1680,46 @@ impl From<Args> for Vec<Metadata> {
         }]
     }
 }
+
+// This type is named the way it is since it appears in docs.  Ints are
+// generally an implementation detail.  But this shouldn't conflict with the
+// `Number` variant of [`KclValue`].
+/// A number that can be either a floating point number or an integer.
+#[derive(Debug, Clone, Copy, JsonSchema, ts_rs::TS, PartialEq)]
+#[ts(export)]
+pub(crate) enum NumberArg {
+    Float(f64),
+    Int(i64),
+}
```

Comment:
This is making me think about the cases where a float will work, but an int will not. Why not use a float for all the cases an int would work? Already our precision cannot go as high as the highest float in many cases. It complicates the usage the numbers IMO.

Reply:
It looks like most code does exactly what you say. It implicitly converts to f64. I did this so that if both your inputs are ints, the result can remain an int. But yeah, since most other things don't do this, so there's little benefit. I guess I'll change it to f64.